diff --git a/dist/get-packages.cjs.dev.js b/dist/get-packages.cjs.dev.js
index 7ebb3c698ee59d776b7e0d3aa2b741d3af41ed67..cdd6eff26febbcbc6e37ae1927eb7da64c21408d 100644
--- a/dist/get-packages.cjs.dev.js
+++ b/dist/get-packages.cjs.dev.js
@@ -232,6 +232,10 @@ function _getPackages() {
             throw new PackageJsonMissingNameError(pkgJsonsMissingNameField);
 
           case 54:
+            if (pkg.name) {
+              results.unshift({ dir: cwd, packageJson: pkg });
+            }
+
             return _context.abrupt("return", {
               tool: tool.type,
               root: {
@@ -359,6 +363,10 @@ function getPackagesSync(dir) {
     throw new PackageJsonMissingNameError(pkgJsonsMissingNameField);
   }
 
+  if (pkg.name) {
+    results.unshift({ dir: cwd, packageJson: pkg });
+  }
+
   return {
     tool: tool.type,
     root: {
